history/characters/wc_easteregg_characters.txt

```diff
@@ -457,4 +457,280 @@ easteregg13={ #Shrek reference
 		}
 	}
 	688.1.1={ death=yes }
-}
\ No newline at end of file
+}
+
+### TALES OF THE PAST MACHINIMA CHARACTERS ###
+
+# Part of Eden Aurorae guild, therefore a landless playable character
+easteregg14 = {
+	name = "Blazer"
+	dna = blazer_dna
+	culture = azerothian
+	religion = holy_light
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 5
+	martial = 5
+	stewardship = 4
+	intrigue = 7
+	learning = 4
+	#Personality
+	trait = brave
+	trait = gregarious
+	trait = stubborn
+	#Education
+	trait = education_intrigue_2
+	#Other
+	trait = adventurer
+	trait = physique_good_1
+	584.3.19 = { 
+		birth = yes
+		trait = creature_human
+	}
+	600.1.1 = {
+		set_relation_friend = character:easteregg15
+	}
+	# switches from lvl 2-3 rogue into paladin around 610, level 3 holy
+	680.6.12 = { death = yes }
+}
+
+# Part of Eden Aurorae guild, therefore a landless playable character
+easteregg15 = {
+	name = "Yimo"
+	dna = yimo_dna
+	culture = gnome
+	religion = rationalism
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 6
+	martial = 4
+	stewardship = 4
+	intrigue = 5
+	learning = 8
+	#Personality
+	trait = compassionate
+	trait = generous
+	trait = diligent
+	#Education
+	trait = education_learning_3
+	#Other
+	trait = adventurer
+	trait = intellect_good_1
+	trait = magic_good_2
+	466.5.10 = { 
+		birth = yes
+		trait = creature_gnome
+	}
+	# level 3 frost mage
+	608.10.14 = { death = { death_reason = death_battle } }	# Killed in Naxxramas
+}
+
+# Tauren mercenary, therefore a landless playable character
+easteregg16 = {
+	name = "Ball"
+	#dna = ball_dna
+	culture = tauren
+	religion = earth_mother_worship
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 3
+	martial = 6
+	stewardship = 4
+	intrigue = 3
+	learning = 3
+	#Personality
+	trait = brave
+	trait = stubborn
+	trait = content
+	#Education
+	trait = education_martial_2
+	#Other
+	trait = adventurer
+	trait = physique_good_1
+	547.8.19 = { 
+		birth = yes
+		trait = creature_tauren
+	}
+	# level 2 warrior
+	662.2.19 = { death = yes }
+}
+
+# Tauren mercenary, therefore a landless playable character
+easteregg17 = {
+	name = "Jubai"
+	#dna = jubai_dna
+	culture = tauren
+	religion = earth_mother_worship
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 5
+	martial = 4
+	stewardship = 3
+	intrigue = 3
+	learning = 6
+	#Personality
+	trait = brave
+	trait = forgiving
+	trait = honest
+	#Education
+	trait = education_learning_3
+	#Other
+	trait = adventurer
+	trait = magic_good_1
+	506.3.10 = { 
+		birth = yes
+		trait = creature_tauren
+	}
+	583.1.1 = {
+		give_nickname = nick_the_ancient
```

Comment:
His ingame nickname is 'Brave' for some reason.
![Στιγμιότυπο οθόνης 2024-09-19 140312](https://github.com/user-attachments/assets/e4a92904-f7cb-476d-887b-005606c2cb08)

Reply:
It seems to be given to many other Tauren for some reason and it overrides everything else for some reason

Reply:
also issue is not related to this branch exclusively